meta/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+galaxy_info:
+  author: Juwai Limited
+  description:
+  company: Juwai Limited
+  license: MIT / BSD
+  min_ansible_version: 1.9
+  platforms:
+  - name: EL
+    versions:
+    - 6
+  - name: Amazon
```

Comment:
/fyi: I commented out the ones on the role branches I've touched to pass the galaxy syntax checker.